packages/overlay/.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
```

Comment:
You need to run `pnpm changeset:add` at project root (not under `packages/overlay`). If you do that, these files won't be generated.